tests/test_common.py

```diff
@@ -0,0 +1,79 @@
+"""Tests for the _common module."""
+
+import pathlib
+import uuid
+
+import pytest
+
+from pysdccc import _common
+
+
+def test_build_command_no_args():
+    """Test that the build_command function works with no arguments."""
+    assert _common.build_command() == []
+
+
+def test_build_command_with_args():
+    """Test that the build_command function works with arguments."""
+    arg1 = uuid.uuid4().hex
+    arg2 = uuid.uuid4().hex
+    assert _common.build_command(arg1, arg2) == [arg1, arg2]
+
+
+def test_build_command_with_args_and_kwargs():
+    """Test that the build_command function works with arguments and keyword arguments."""
+    arg1 = uuid.uuid4().hex
+    arg2 = uuid.uuid4().hex
+    key1 = uuid.uuid4().hex
```

Comment:
This are the values and not the keys, arent they?

Reply:
done